frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -32,6 +32,7 @@
         vm.isChallengeHost = false;
         vm.stopLeaderboard = function() {};
         vm.stopFetchingSubmissions = function() {};
+        vm.currentDate = new Date().toISOString();
```

Comment:
@Ram81 Please use the time from the server as both front-end and back-end are decoupled.

Reply:
Sure @RishabhJain2018 will do it.

Reply:
@RishabhJain2018 is there an api already available to get server's current time ?

Reply:
No, there isn't any API for that.